src/adapters/handlers/items.ts

```diff
@@ -53,15 +53,20 @@ export function createItemsHandler(
       'emotePlayMode',
       EmotePlayMode
     )
+    const ids = params.getList('id')
     const contractAddresses = params.getList('contractAddress')
     const itemId = params.getString('itemId')
     const network = params.getValue<Network>('network', Network)
     const maxPrice = params.getString('maxPrice')
     const minPrice = params.getString('minPrice')
     const urns = params.getList('urn')
 
-    return asJSON(() =>
-      items.fetchAndCount({
+    return asJSON(() => {
+      if (ids && (contractAddresses || itemId || urns)) {
```

Comment:
```suggestion
      if (ids.length && (contractAddresses.length || itemId || urns.length)) {
```

As empty arrays are truthy and will always return true no matter the size